### PR TITLE
workflows: docs publishing

### PR DESCRIPTION
**Pull request by woodruffw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +29,7 @@ python -m pip install stdlib-list
 ['AL', 'BaseHTTPServer', 'Bastion', 'CGIHTTPServer', 'ColorPicker', 'ConfigParser', 'Cookie', 'DEVICE', 'DocXMLRPCServer', 'EasyDialogs']
 ```
 
-For more details, check out [the docs](http://python-stdlib-list.readthedocs.org/en/latest/).
```
Comment (miketheman) [inline README.md:28]: Has this been deactivated on RTD?

Comment (woodruffw) [reply README.md:28]: I don't think it's been deactivated, but I'm also not too familiar with RTD: it looks like it's hosting a copy of the docs build from the repo pre-transfer, so maybe the RTD ownership needs to be transferred manually?

(Right now it's being rendered here: <https://pypi.github.io/stdlib-list/>, but that doesn't include version history.)

Comment (miketheman) [reply README.md:28]: > maybe the RTD ownership needs to be transferred manually

yes - or shut down/removed, as you're opting for GH Pages now.

Comment (woodruffw) [reply README.md:28]: > yes - or shut down/removed, as you're opting for GH Pages now.

Cool, I'll reach out to the original maintainer and see if they can remove it.

Comment (woodruffw) [reply README.md:28]: Original maintainer has removed the old RTD project.

**Pull request merged**